[2021-08-12 10:53:55,171] {taskinstance.py:886} INFO - Dependencies not met for <TaskInstance: parallel_dag_02.task_1 2021-08-11T00:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state which is not a valid state for execution. The task must be cleared in order to be run.
[2021-08-12 10:53:55,172] {local_task_job.py:96} INFO - Task is not able to be run
[2021-08-12 11:05:22,979] {taskinstance.py:886} INFO - Dependencies not met for <TaskInstance: parallel_dag_02.task_1 2021-08-11T00:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state which is not a valid state for execution. The task must be cleared in order to be run.
[2021-08-12 11:05:22,979] {local_task_job.py:96} INFO - Task is not able to be run
